app/editor/src/features/reports/av-overview/AVOverview.tsx

```diff
@@ -86,9 +85,12 @@ const AVOverview: React.FC = () => {
       <Row className="page-header">
         <h1>Evening Overview</h1>
         <div className="buttons">
```

Comment:
You can replace the `{' '}` by simply updating the styling for "buttons".  Add `gap: 0.5rem` to the buttons style and make them a flexbox row.

Reply:
Updated, thanks